tests/integration_tests/bugs/test_gh868.py

```diff
@@ -0,0 +1,19 @@
+"""Ensure no Traceback when 'chef_license' is set"""
+import pytest
+from tests.integration_tests.instances import IntegrationInstance
+
+
+USERDATA = """\
+#cloud-config
+chef:
+  install_type: omnibus
```

Comment:
Are there any concerns about this being in an integration test? This causes chef to install from `https://www.chef.io/chef/install.sh`

Reply:
I like this test because it tests a realworld application/use-case all the way to the source.

All good to have the remote script source. I don't mind this for two reasons:
 - security: not concerned as it's a short-lived instance that is torn down immediately after the test run
 - scripts on the internet changing which break our tests: let's deal/cope with that only if we find that this test is really flaky due to changes in the script.
 
Are there other concerns you had in mind?

Reply:
Just from a "good steward" point of view, but I can't image our jenkins runs would be more than a blip compared to the number of hits it receives.

Only reason I thought of it was because we have an older cloud test for chef that is disabled because it relies on a 3rd party repo. https://github.com/canonical/cloud-init/blob/master/tests/cloud_tests/testcases/examples/install_run_chef_recipes.yaml#L4

Reply:
I'm okay with us keeping this for now. That commit where we disabled was pointing to https://bugs.launchpad.net/cloud-init/+bug/1678145 which was basically a "don't let things on the internet break us when they change". If this ends up breaking repeatedly and causing maintenance problems, We can spend time to remedy/cache it or something else.